script.js

```diff
@@ -0,0 +1,59 @@
+const savedList = document.getElementById('savedList');
+const doneList = document.getElementById('doneList');
+let savedListTitle = document.getElementById('savedListTitle');
+savedListTitle.innerHTML = '해야할 일';
+doneListTitle.innerHTML = '완료한 일';
+
+form.addEventListener('submit', (e) => {
+  //할 일 개수 세기
+  const countSavedNum = () => {
+    let savedNum = savedList.getElementsByTagName('li').length;
+    savedListTitle.innerHTML = `해야할 일(${savedNum})`;
+  };
+
+  const countDoneNum = () => {
+    let doneNum = doneList.getElementsByTagName('li').length;
+    doneListTitle.innerHTML = `완료한 일(${doneNum})`;
+  };
+
+  //할 일 구현
+  const li = document.createElement('li');
+  li.innerText = document.getElementById('input').value;
+  li.id = li.innerText;
+  li.isDone = false;
+  li.isDone ? doneList.append(li) : savedList.append(li);
+  countSavedNum();
+
+  //삭제 버튼
+  const deleteButton = document.createElement('button');
+  deleteButton.innerText = 'x';
+  deleteButton.id = 'deleteButton';
+  li.append(deleteButton);
+
+  deleteButton.addEventListener('click', function (event) {
+    setTimeout(() => li.remove(), 0);
+  });
+
+  //할 일이 완료 되었는지 확인
+  li.addEventListener('click', () => {
+    li.isDone ? (li.isDone = false) : (li.isDone = true);
```

Comment:
```suggestion
    li.isDone = !li.isDone;
```
로 표현할 수도 있는것 참고하시면 좋을것 같아요!

Reply:
제가 몰랐던 부분이네요!! 혹시 이 문법은 어떻게 검색하면 더 알아볼 수 있을까요?

Reply:
정리된 검색결과를 찾으려고 했는데 못 찾았네요 ㅠㅠ
다른 방법으로도 작성할 수 있다고 말씀 드린것 뿐입니다! 
코딩 스타일이라 민지님 처럼 작성하셔도 아무런 문제가 없을것 같아요~

Reply:
[not 연산자](https://hanamon.kr/javascript-logical-not-operator/) 찾아 보시면 될 거예요~

Reply:
링크까지!! 알려주셔서 감사드립니다🙌

Reply:
not 연산자를 말씀하시는거 였군요! 제가 질문을 잘못 이해했었네요 😅